src/components/molecules/LiveSchedule/LiveSchedule.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FC, useMemo } from "react";
+
+import { isEventLive } from "utils/event";
+import { venueEventsSelector } from "utils/selectors";
+
+import { useSelector } from "hooks/useSelector";
+
+import "./LiveSchedule.scss";
+
+import { LiveEvent } from "./LiveEvent";
+
+const LiveSchedule: FC = () => {
+  const venueEvents = useSelector(venueEventsSelector);
+
+  const liveEvents = useMemo(() => {
+    return venueEvents && venueEvents.length
+      ? venueEvents.filter((event) => isEventLive(event))
+      : [];
+  }, [venueEvents]);
+
+  const hasLiveEvents = !!liveEvents.length;
+
+  if (!hasLiveEvents) {
+    return <div className="schedule-event-empty">No live events for now</div>;
+  }
+
+  return (
+    <div className="schedule-container show">
+      <div className="schedule-day-container">
+        {liveEvents.map((liveEvent, index) => {
+          return (
+            <LiveEvent key={`live-event-${index}`} liveEvent={liveEvent} />
+          );
+        })}
```

Comment:
[NB, nit, stylistic] If the block is short and clean like this, you can drop the `{ }` and explicit `return` and just use the `=>` directly

eg. 
```suggestion
        {liveEvents.map((liveEvent, index) => <LiveEvent key={`live-event-${index}`} liveEvent={liveEvent} />)}
```

No need to worry about it for this PR though, more FYI.